src/changelog.md

```diff
@@ -0,0 +1,33 @@
+
+# Release 2.2.0
+* Requires AzureStack Update 2108+
+* New module Azs.ContainerRegistry.Admin has been included exposing the apis to setup container registry in AzureStack
+* New module Azs.ContainerService.Admin has been included exposing quota APIs
```

Comment:
Could we please modify release notes to "AzureStack AKS admin power shell commands to manage    Container service admin operations"

Reply:
Could we also change the release notes to "AzureStack Container Registry admin power shell commands to setup and manage Container Registry admin operations"